web/reports/etv.py

```diff
@@ -333,8 +333,12 @@ def __return_all(self):
         while self.scope.can_return():
             self.__return(if_possible=True)
 
-    def __get_comment(self, note, warn, env):
+    def __get_comment(self, note, warn, env, env_relevant):
```

Comment:
I suggest to unify code for processing all arguments `note, warn, env, env_relevant`.